pkg/content/file.go

```diff
@@ -28,6 +28,9 @@ type FileStore struct {
 	DisableOverwrite          bool
 	AllowPathTraversalOnWrite bool
 
+	// Reproducible enables stripping times from added files
+	Reproducible bool
```

Comment:
Maybe `StripTimes` is better?

Reply:
@shizhMSFT I decided to make it more generic, in case there are more things to strip